.github/workflows/main.yml

```diff
@@ -154,6 +154,27 @@ jobs:
           name: Windows Artifacts
           path: Binaries/*.zip
 
+  analyze-uncrustify:
+    name: Analyze Uncrustify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install curl git unzip
+
+      - name: CI Bootstrap
+        run: |
+          src=$(/usr/bin/curl -Lfs https://raw.githubusercontent.com/acidanthera/ocbuild/master/ci-bootstrap.sh) && eval "$src" || exit 1
+
+      - name: Run Uncrustify
+        run: |
+          src=$(curl -Lfs https://raw.githubusercontent.com/acidanthera/ocbuild/uncrustify/uncrustify/uncstrap.sh) && eval "$src" || exit 1
```

Comment:
TODO: Switch to master when https://github.com/acidanthera/ocbuild/pull/15 gets merged.